Chinese/congrats/congrats.md

```diff
@@ -1,2 +1,2 @@
 恭喜您已經被選中參加我們的年度訪客意見調查。
-完成後，您將有機會獲得一台 Macbook Air, iPhone 7, 或一台 iPad Air 3.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.